### PR TITLE
Updated Translations

### PR DESCRIPTION
**Pull request by devendrapoonia**
Removed incorrect translations, and added new translations.

### CODE REVIEW

File: src/_locales/hi/messages.json [hunk 1]
```diff
@@ -1,72 +1,72 @@
 {
 	"appDesc": {
-		"message": "अपना समय बर्बाद ना करें। यूनिवर्सल बायपास आप को तंग करने वाले शॉर्टर्स लिंक से निजात दिलाता है।"
+		"message": "अनुपालना में अपना समय बर्बाद ना करें। FastForward कष्टप्रद लिंक शॉर्टंरों को रोकता है।"
```
Comment (NotAProton) [inline src/_locales/hi/messages.json]: शॉर्टंरों is a bit hard to understand wouldn't "shorteners"(english)  be better?

File: src/_locales/hi/messages.json [hunk 2]
```diff
@@ -75,84 +75,84 @@
...
 	"cancel": {
-		"message": "रद्द करें"
+		"message": "रद्द करें।"
 	},
 	"crowdBypassedIgnore": {
 		"message": "क्या ये सही नहीं था?"
 	},
 	"crowdBypassedOptions": {
-		"message": "क्या आप चाहते हैं कि यह तुरंत एक नए टैब में या एक्स सेकंड में खोला जाए?"
+		"message": "क्या आप चाहते हैं कि यह तुरंत एक नए टैब में या x सेकंड में खोला जाए?"
```
Comment (NotAProton) [inline src/_locales/hi/messages.json:150]: x ki jagah "कुछ" ?

Comment (devendrapoonia) [reply src/_locales/hi/messages.json:150]: Nope.

File: src/_locales/hi/messages.json (hunk 1 above)
Comment (devendrapoonia) [reply src/_locales/hi/messages.json]: Yeah, you are right, This would be much better - 'शॉर्टनरों'

File: src/_locales/hi/messages.json (hunk 2 above)
Comment (devendrapoonia) [reply src/_locales/hi/messages.json:150]: This 'x' will be replaced with something as I know.

Comment (NotAProton) [reply src/_locales/hi/messages.json:150]: no it's actually just the letter 'x' on the before navigate page 
image

Comment (devendrapoonia) [reply src/_locales/hi/messages.json:150]: it shows change in FastForward options, so if that setting is changed with any number then 'x' will be replaced by that number.
image

I think this setting affects this 'x'

Comment (devendrapoonia) [reply src/_locales/hi/messages.json:150]: Thanks for correcting me, I changed replaced 'x' with 'कुछ'. I hope there is no any other issue in file, if there is any then let me know.

**Review by NotAProton — APPROVED** [review]

**Pull request merged**